lib/services/elasticsearch.js

```diff
@@ -1406,7 +1389,7 @@ class ElasticSearch extends Service {
    *
    * @return {Promise.<Object>} { items, errors }
    */
-  mCreateOrReplace(
+  async mCreateOrReplace(
```

Comment:
Why making the method `async` even if it didn't use `await` or `new Promise` ?

Reply:
Because there are assertions that can throw exception. What we expect from that method is to return a promise (fulfilled or rejected), we don't want it to throw